getting-started/src/main/kotlin/com/google/cloud/kotlin/demo/EmojifyController.kt

```diff
@@ -83,18 +83,21 @@ fun downloadObject(objectName: String) {
     writeTo.close()
 }
 
+data class EmojifyResponse(val url: String)
```

Comment:
Why a data class?

Reply:
I think a normal class should be enough. Changing this to a simple class